website_docs/getting_started/browser.md

```diff
@@ -43,13 +43,8 @@ Copy the following file into an empty directory and call it `index.html`.
 To create traces in the browser, you will need `@opentelemetry/web`, and the plugin `@opentelemetry/plugin-document-load`:
 
 ```shell
-npm install @opentelemetry/web @opentelemetry/plugin-document-load
-```
-
-In the following we will use parcel as web application bundler, but you can of course also use any other build tool:
-
-```shell
-npm install -g parcel
+echo '{}' > package.json
```

Comment:
I'd prefer this to be an `npm init -y`, since this echo is destructive if the user has `package.json` already. It also is non conformant to the (albeit loose) package.json spec which requires at least a `name` and `version` field.
```suggestion
npm init -y
```

Reply:
Thanks & fixed!